spec/lib/utils/job-utils/redfish-tool-spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node: true */
+
+'use strict';
+
+describe("redfish-tool", function() {
+    var redfishTool;
+    
+    before(function() {
+         helper.setupInjector([
+            helper.require('/lib/utils/job-utils/redfish-tool.js'),
+        ]);
+        redfishTool = helper.injector.get('JobUtils.RedfishTool');
+    });
```

Comment:
There is no need to back up the value again in these unit-tests.  Other tests relying on TLS (which seems wrong for a unit test to begin with) should set this variable accordingly.